root/components/CommonsImageWrapper.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import * as manifest from '../static/manifest';
+
+import CommonsImage from './CommonsImage';
+import Frag from './Frag';
+
+type Props = {|
+  +image?: CommonsImageT | null,
+  +imageEndpoint?: string,
+|};
+
+const CommonsImageWrapper = (props: Props) => (
+  <Frag>
+    <div id="commons-image">
+      <CommonsImage image={props.image} />
+    </div>
+    {props.image ? null : manifest.js('commons-image', {
```

Comment:
Hm, not really, because I didn't know about that requirement when I wrote that code. :)

We needed `div#commons-image` to always exist because that's the mount point. The mount point has to exist because the hydrate function requires a root node.

I also could have wrote:

```JSX
<div id="commons-image">
  {props.image ? <CommonsImage image={props.image} /> : null}
</div>
```

But `<CommonsImage />` already renders to `null` in that case, so the check was unnecessary.